src/recipe.rst

```diff
@@ -19,6 +19,29 @@ Your final recipe should have no comments and follow the order in the example.
 *If there are details you are not sure about please open a pull request. The conda-forge team will be happy to answer your questions.*
 
 
+In case you build your first recipe using conda-forge, the we provide here a step-by-step instruction and checklist that might help you with a sucessfully build
+
+Step-by-step instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. fork the `example recipes <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_
+2. within your forked copy, generate a new folder in the recipes subdirectory and copy meta.yml from the example directory. Leave the example directory unchanged!
+3. edit the example recipe (`meta.yml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml>`_) as you need it
+4. generate the ssh 256 key like described in the example recipe using the
+   openssl tool
```

Comment:
yes, but only from the new web interface and I had some trouble to find it for some packages. That's why I did not integrate it so far